components/WrapperCanvas.vue

```diff
@@ -1,16 +1,25 @@
 <script setup>
+    import Comic from '~/utils/Classes/Comic';
+    import ElementDS from '~/utils/Classes/Element';
+    import Text from '~/utils/Classes/Text';
+    import Asset from '~/utils/Classes/Asset';
+
     const props = defineProps({
         height: Number,
         panel: Object,
-        selectedId: String,
+        panelIsActive: Boolean,
+        comic: Object,
         lockAspectRatio: Boolean,
+        panelIndex: Number,
+        arrayBB: Object,
     });
 
-    const canvasHeight = computed(() => props.height + 'px');
-    const canvasWidth = computed(() => props.panel.currentState().width + 'px');
+    const canvasHeight = computed(() => props.arrayBB[props.panelIndex].height + 'px');
+    const canvasWidth = computed(() => props.arrayBB[props.panelIndex].width + 'px');
```

Comment:
Wouldn't it be better to just pass the element for the active panel into the component.

Reply:
I even think you can just get the size of the panel from in this component, if you give the wrapper div a ref. Since the `swiper-no-swiping`-class would just be added to this div.